### ISSUE TITLE
Decreased cardiac output

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6562]
**Preferred term label:** Decreased cardiac output

**Synonyms**
Decreased heart output

**Definition (free text, please give PubMed ID)**
A decreased volume of blood pumped by the left and right ventricle, per unit time. Cardiac output (CO) is the product of the heart rate (HR), i.e. the number of heartbeats per minute (bpm), and the stroke volume (SV), which is the volume of blood pumped from the ventricle per beat. 

**Parent term (use hpo.jax.org/app)**
Abnormality of blood circulation HP:0011028

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
